firebase-functions/webhook-router/src/express.d.ts

```diff
@@ -0,0 +1,11 @@
+import type { Region } from "./webookConfig";
+
+declare global {
+  namespace Express {
+    interface Request {
+      regions?: Region[];
```

Comment:
Needed by typescript to pass in regions from verification middleware to forwarder